web-api/tests/semantic_search.rs

```diff
@@ -351,3 +375,46 @@ fn test_semantic_search_with_dev_option_candidates() {
         },
     );
 }
+
+#[test]
+fn test_semantic_search_with_snippets() {
```

Comment:
:art: we should indicate in the name that this is a dev option like we do above
```suggestion
fn test_semantic_search_with_dev_option_snippet() {
```